solutions/devsprint-1/DeliveryAppChallenge/Components/DividerView.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  DividerView.swift
+//  DeliveryAppChallenge
+//
+//  Created by Gabriel on 03/11/21.
+//
+
+import Foundation
+import UIKit
+
+public class DividerView: UIView {
+    
+     lazy var dividerView: UIView = {
```

Comment:
Como não estamos (e também não será necessário) acessando esse elemento fora dessa classe, podemos manter como privado.
```suggestion
     private lazy var dividerView: UIView = {
```